bootstrap-joining-demo/marathon-api/src/main/scala/akka/cluster/bootstrap/DemoApp.scala

```diff
@@ -5,14 +5,41 @@
 package akka.cluster.bootstrap
 
 import akka.actor.ActorSystem
+import akka.cluster.{ Cluster, MemberStatus }
+import akka.http.scaladsl.Http
+import akka.http.scaladsl.model._
+import akka.http.scaladsl.server.Directives._
 import akka.management.AkkaManagement
 import akka.management.cluster.bootstrap.ClusterBootstrap
+import akka.stream.ActorMaterializer
 
 object DemoApp extends App {
-
   implicit val system = ActorSystem("my-system")
+  implicit val materializer = ActorMaterializer()
+
+  val cluster = Cluster(system)
+
+  def isReady() = {
+    val selfNow = cluster.selfMember
+
+    selfNow.status == MemberStatus.Up
+  }
+
+  def isHealthy() = {
+    isReady()
+  }
+
+  val route =
+    concat(
+      path("ping")(complete("pong!")),
+      path("healthy")(complete(if (isHealthy()) StatusCodes.OK else StatusCodes.ServiceUnavailable)),
+      path("ready")(complete(if (isReady()) StatusCodes.OK else StatusCodes.ServiceUnavailable)))
```

Comment:
Additionally, we need to modify, application.conf

akka {

  discovery.method = marathon-api

  actor {
    provider = "cluster"
  }

  remote {
    netty.tcp {
      port = 2551
    }
  }

  management {
    http {
      port = 19999
    }
  }

  management {

    cluster.bootstrap {

      contact-point {
        fallback-port = 19999

        no-seeds-stable-margin = 5 seconds
      }

    }

  }

}